### PR TITLE
Adding label 24 / parsing

### PR DESCRIPTION
**Pull request by makrsmark**

### CODE REVIEW

File: lib/plugins/Label_24_Slash.test.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    const decodeResult = plugin.decode({ text: text });
+    expect(decodeResult.decoded).toBe(true);
+    expect(decodeResult.decoder.decodeLevel).toBe('partial');
+    expect(decodeResult.raw.message_timestamp).toBe(1729160460);
+    expect(decodeResult.raw.flight_number).toBe('04WM');
+    expect(decodeResult.raw.altitude).toBe(34962);
+    expect(decodeResult.raw.position.latitude).toBe(53.13);
+    expect(decodeResult.raw.position.longitude).toBe(1.33);
+    expect(decodeResult.raw.eta_time).toBe(39360);
+    expect(decodeResult.formatted.items.length).toBe(3);
```
Comment (makrsmark) [inline lib/plugins/Label_24_Slash.test.ts:35]: went a little different with the specs - assuming the formatted will be fine. not sure how i feel about it

Comment (kevinelliott) [reply lib/plugins/Label_24_Slash.test.ts:35]: Interesting. I suppose it makes the spec simpler, but I suppose we won't know that formatted results have changed accidentally.

File: lib/plugins/Label_24_Slash.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      ResultFormatter.position(decodeResult, position);
+      ResultFormatter.eta(decodeResult, DateTimeUtils.convertHHMMSSToTod(fields[8]+'00'));
+      decodeResult.remaining.text = fields[7];
+
+      decodeResult.decoded = true;
+      decodeResult.decoder.decodeLevel = 'partial';
+    } else {
+      // Unknown!
+      if(options.debug) {
+        console.log(`DEBUG: ${this.name}: Unknown variation. Field count: ${fields.length}`);
```
Comment (kevinelliott) [inline lib/plugins/Label_24_Slash.ts:46]: Might we want to output the remaining here?

**Review by kevinelliott — APPROVED** [review]
LGTM, nothing blocking, just some comments.

File: lib/plugins/Label_24_Slash.test.ts (hunk 1 above)
Comment (makrsmark) [reply lib/plugins/Label_24_Slash.test.ts:35]: Added until there's specific tests for ResultFormatter

File: lib/plugins/Label_24_Slash.ts (hunk 2 above)
Comment (makrsmark) [reply lib/plugins/Label_24_Slash.ts]: Added the message back in

**Pull request merged**